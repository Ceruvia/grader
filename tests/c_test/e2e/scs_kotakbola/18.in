540 952 186
H 60
B 57
H 12
H 24
M 65
M 13
H 100
M 84
M 27
M 3
B 68
H 55
M 19
M 82
B 24
B 5
H 82
M 30
H 62
E
