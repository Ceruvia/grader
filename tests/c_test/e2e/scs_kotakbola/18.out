395 323 154
Ya
